src/mem/pgtable.rs

```diff
@@ -10,7 +10,7 @@ use crate::cpu::vc::*;
 use crate::cpu::vc_make_page_private;
 use crate::cpu::vc_make_page_shared;
 use crate::cpu::vc_terminate;
-use crate::dynam::mem_allocate_frame;
+use crate::mem::mem_allocate_frame;
```

Comment:
This breaks rustfmt, rearrange following rustfmt --check. You can also just do rustfmt src/lib.rs